### ISSUE TITLE
Please write comments

### ISSUE THREAD

**Issue by funwarioisii** [Steppschuh/Sensor-Data-Logger#2]
Firstly, I appreciate that you made this app!

I wanna take some sensor values and write out them to any files.
And I start to develop the part of file writer, but you leave few comments.
So, I couldn't read your code well. Sorry.

Please write comments to some classes.

**Comment by glazik12** [comment]
@hashimock any news about file writer?

**Comment by dufferzafar** [comment]
Yeah, having exporting functionality is all that is missing from this app!

**Comment by 549033790** [comment]
I really appreciate that you made this app, but I have some problem when I use this app. Would you please tell me how to link the watch and my phone, the app only display the sensor message on my phone, thank you very much

**Comment by Steppschuh** [comment]
Please create a separate issue for that, @549033790. Also, try to include some more detailed information about what you did and what you expected to happen.

**Comment by chazhenry** [comment]
I'm running the mobile app on a Samsung Note that is paired to a Ticwatch E. The watch is running the WearOS app. When i start the phone app, it sees that its paired and tells me to start the DataLogger on the Ticwatch E. It clearly knows it's paired as it calls the watch by name.

However, when I run sensordatalogger.apk on the watch, the phone app still only shows and reports "Sensors on this device". How do I get the phone app to show watch sensor data? Thanks!
